src/components/interactive-builder/modals/question/question-form/rendering-types/inputs/workspace-launcher/workspace-launcher.component.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { TextInput } from '@carbon/react';
+import { useFormField } from '../../../../form-field-context';
+
+const WorkspaceLauncher: React.FC = () => {
+  const { t } = useTranslation();
+  const { formField, setFormField } = useFormField();
+
+  return (
+    <>
+      <TextInput
+        id="buttonLabel"
+        labelText={t('buttonLabel', 'Button Label')}
+        value={formField.questionOptions?.buttonLabel ?? ''}
+        onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+          const updatedQuestion = {
+            ...formField,
+            questionOptions: { ...formField.questionOptions, buttonLabel: event.target.value },
+          };
+          setFormField(updatedQuestion);
```

Comment:
Please extract this into its own function and wrap it in a useCallback - https://o3-docs.openmrs.org/docs/coding-conventions/performance
```
Avoid inline function definitions in JSX
// Bad - new function created every render
<Button onClick={() => handleClick(id)} />
 
// Good - stable function reference
const handleButtonClick = useCallback(() => {
  handleClick(id);
}, [id, handleClick]);
 
<Button onClick={handleButtonClick} />
```